index.js

```diff
@@ -21,7 +21,7 @@ class Verifier {
   constructor(params, claims = {}) {
     if (!params.userPoolId) throw Error('userPoolId param is required');
     if (!params.region) throw Error('region param is required');
-    if (!params.debug) {
+    if (params.debug === undefined) {
```

Comment:
How about we just set `this.debug = true` in the constructor.  And then  `if (params.debug === false) { this.debug = false; }

Reply:
Sounds good.

Reply:
Updated this MR with your suggestion.